packages/block-library/src/page-list/convert-to-links-modal.js

```diff
@@ -5,14 +5,14 @@ import { Button, Modal } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 
 export const convertDescription = __(
-	'This menu is automatically kept in sync with pages on your site. You can manage the menu yourself by clicking "Edit" below.'
+	'This page list is synced with the published pages on your site. Unsync the page list to add, delete, or reorder pages yourself.'
```

Comment:
Should we use "detach" here instead of unsync? To match patterns and template parts.
https://github.com/WordPress/gutenberg/issues/51940#issuecomment-1608431204
https://github.com/WordPress/gutenberg/pull/51993